lightbeam/test.py

```diff
@@ -0,0 +1,214 @@
+import os
+import json
+
+class Tester:
+
+    def __init__(self, lightbeam=None):
+        self.lightbeam = lightbeam
+        self.logger = self.lightbeam.logger
+    
+    def test(self, tests_dir):
+        self.lightbeam.results = []
+
+        # * Test `lightbeam count`:
+        self.logger.info("testing `lightbeam count`...")
+        count_file = os.path.join( tests_dir, "output", "count.tsv" )
+        self.lightbeam.results_file = count_file
+        self.lightbeam.selector = "localEducationAgencies,schoolTypeDescriptors"
```

Comment:
I use `localEducationAgencies` and `schoolTypeDescriptors` because (1) they're small (but non-empty) in the Grand Bend dataset and (2) they (and their structure) are unlikely to change with data standards changes.